### PR TITLE
fix(dd-rum): add missing host for connect-src

### PR DESCRIPTION
**Pull request by stanleynguyen**
Add missing CSP headers for RUM to record sessions

### CODE REVIEW

File: packages/backend/src/helpers/csp.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ const helmetOptions: HelmetOptions = {
       connectSrc: [
         "'self'",
         // For Datadog RUM
+        'https://browser-intake-datadoghq.com',
         'https://*.browser-intake-datadoghq.com',
```
Comment (ogp-weeloong) [inline packages/backend/src/helpers/csp.ts:15]: Do we still need this?

Comment (ogp-weeloong) [reply packages/backend/src/helpers/csp.ts:15]: Not a priority to resolve now btw, just wondering. we can test another time.

Comment (stanleynguyen) [reply packages/backend/src/helpers/csp.ts:15]: yes we actually need this, they changed the hostname for RUM

**Review by ogp-weeloong — APPROVED** [review]
LGTM

File: packages/backend/src/helpers/csp.ts [hunk 2]
```diff
@@ -39,6 +40,7 @@ const helmetOptions: HelmetOptions = {
         "'self'",
         'https://www.google-analytics.com',
         'https://www.googletagmanager.com',
+        'https://www.datadoghq-browser-agent.com',
```
Comment (ogp-weeloong) [inline packages/backend/src/helpers/csp.ts]: can remove if not needed, i dont think we will ever be using CDN version

**Pull request merged**